docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
-Shows a list of all persons in the address book.<br>
-Format: `list`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+#### Updating a task : `update`
+Updates a task from the list displayed<br>
+Format: `update todo/schedule INDEX description/date/start/end/status/priority  NEW_VALUE`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * Updates the description/date/start/end/status/priority of the task at the specified `INDEX`<br> from todo tasks or schedule as specified.<br> 
+> * The index refers to the index number shown in the current listing.<br>
+> * The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `update todo 3 description Avengers`<br>
+   Selects the 3rd todo task from the displayed list and changes the task description to �Avengers�
+* `update todo 4 date 11 Nov`<br>
+   Selects the 4th todo task from the displayed list and changes the type of task from todo to scheduled. 
+* `update schedule 5 time 8:30pm`<br>
+   Selects the 5th schedule task from the displayed list and changes the time to 8:30pm. <br>
+<img src="images/UI_Prototype/WhatNowUI-UpdateSchedule-Colour.png" width="600"> 
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+#### Undoing the previous action : `undo`
+There can be multiple undos to revert to the previous state.<br>
+Format: `undo`  
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+* `undo`<br>
+   Undo the previous action <br>
+<img src="images/UI_Prototype/WhatNowUI-Undo-Colour.png" width="600">
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+#### Redoing the previous action : `redo`
+There can be multiple redos to revert to the previous state.<br>
```

Comment:
1) If you redo without an undo?

It will not change anything and it will display in the feedback box that there are nothing to be redone something like this. We will think of how to phrase it.

2) If you redo multiple times after undo?

Same as above when there is nothing to redo.